prac_05/hex_colours.py

```diff
@@ -0,0 +1,24 @@
+def main():
+    colour_to_hex = {"Absolute Zero": "#0048ba",
```

Comment:
Your colour names need to be in all capitals or all lowercase. Otherwise, you won't be able to make them case insensitive.